### DISCUSSION TITLE
issue with new Endpoint security policies

### DISCUSSION THREAD

**Discussion by bigmsdn** [Micke-K/IntuneManagement#75] [Q&A]
New Endpoint Security policies (those which support filters, eg : Antivirus policy) are not displayed in Intune Manager.

**Comment by Micke-K** [comment]
Hello!

Did you look under the Settings Catalog node?

Intune uses the Settings Catalog API for some of the Endpoint Security policies. 

Cheers!

**Comment by bigmsdn** [comment]
Hello and thank you for your quick answer ! And for this awesome app 👍
You’re right : They have changed something for these new policies (which support filters) : I found them in settings catalog on Intune Manager .

Best Regards


Envoyé à partir de Outlook<http://aka.ms/weboutlook>
________________________________
De : Mikael Karlsson ***@***.***>
Envoyé : mercredi 3 août 2022 10:21
À : Micke-K/IntuneManagement ***@***.***>
Cc : bigmsdn ***@***.***>; Author ***@***.***>
Objet : Re: [Micke-K/IntuneManagement] issue with new Endpoint security policies (Discussion #75)


Hello!

Did you look under the Settings Catalog node?

Intune uses the Settings Catalog API for some of the Endpoint Security policies.

Cheers!

**Reply by Micke-K** [reply]
Thank you so much for the support. I really appreciate it!

Cheers
